app/routes/jobs/new/build-answer-set.js

```diff
@@ -10,7 +10,7 @@ export default Ember.Route.extend({
     };
     return Ember.RSVP.hash(promises).then(hash => {
       return store.createRecord('job-answer-set', {
-        jobName: 'FIXME',
+        jobName: '',
```

Comment:
Can we remove this line then, if the blank value is never valid? Or does the control need it bound to `''`?